testing/arkane/two_parameter_arrhenius_fit/impiriamine.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+energy = Log('impiriamine_sp.log')
```

Comment:
should this be `imipramine`? (also the file name)

Reply:
thanks for catching this